storybook/stories/goal/2_gauge_with_target.story.tsx

```diff
@@ -70,10 +70,11 @@ export const Example = () => {
         bandFillColor={useColors ? ({ value }: BandFillColorAccessorInput) => bandFillColor(value) : undefined}
         labelMajor="Revenue 2020 YTD  "
         labelMinor="(thousand USD)  "
-        centralMajor={`${actual}`}
+        centralMajor={() => `$${actual}k USD`}
         centralMinor=""
         angleStart={angleStart}
         angleEnd={angleEnd}
+        tooltipValueFormatter={(d) => `${d}k USD`}
```

Comment:
Did you want the `$` here too?

```suggestion
        tooltipValueFormatter={(d) => `$${d}k USD`}
```

Reply:
thank you! 9e61b47